doc/qbk/1.0.overview.qbk

```diff
@@ -140,8 +146,12 @@ informal works. In this library we use the term [*URL] to
 refer to all strings which are valid according to the
 top-level grammar rules found in __rfc3986__.
 
+[$url/images/MidwitMeme.svg]
```

Comment:
I wouldn't include that in the docs, but well...

Reply:
I will remove it before releasing to Boost